app/models/menu/menu.rb

```diff
@@ -471,7 +471,7 @@ def warehouse_status_menu
         title: 'Imports',
       ),
     )
-    if RailsDrivers.loaded.include?(:ma_reports)
+    if RailsDrivers.loaded.include?(:ma_reports) && MaReports::CsgEngage::Credential.present?
```

Comment:
Can we limit this to active credentials? (and thanks!)
```suggestion
    if RailsDrivers.loaded.include?(:ma_reports) && MaReports::CsgEngage::Credential.active.present?
```